src/main/java/org/hawkular/client/android/activity/AuthorizationActivity.java

```diff
@@ -98,7 +87,11 @@ public void setUpAuthorization() {
             return;
         }
 
-        setUpBackendAuthorization();
+        Intent login_activity = new Intent(getApplicationContext(), LoginActivity.class);
```

Comment:
Sir you might have not noticed but here Authz is not being used and is continued only coz of sustaining next functionalities. From discussion with summers and suggestion of jpkrohling. 
Authz cant provide feature of using key/secret